.teamcity/components/settings.kt

```diff
@@ -144,7 +144,7 @@ var serviceTestConfigurationOverrides = mapOf(
         "networkfunction" to testConfiguration(locationOverride = LocationConfiguration("westus2", "eastus2", "westeurope", false)),
 
         // Network Regional Tire Public IP is only available in
-        "network" to testConfiguration(locationOverride = LocationConfiguration("westeurope", "eastus2", "westus", false)),
+        "network" to testConfiguration(locationOverride = LocationConfiguration("eastus", "westus", "westeurope", false)),
```

Comment:
Why the need to change the region? We keep to `westeurope` as the Primary unless there's a specific reason not to.

Reply:
Seems it was unstable in this region. OK. I will change it back.